coldfront/core/allocation/urls.py

```diff
@@ -95,13 +95,10 @@
         path('<int:pk>/secure-dir-<str:action>-user-deny-request',
              secure_dir_views.SecureDirManageUsersDenyRequestView.as_view(),
              name='secure-dir-manage-user-deny-request'),
-        path('secure-dir-request-landing/',
-             TemplateView.as_view(
-                 template_name=
-                 'secure_dir/secure_dir_request/secure_dir_request_landing.html'
-             ),
+        path('secure-dir-request-landing/<str:project_name>',
+             secure_dir_views.SecureDirRequestLandingView.as_view(),
              name='secure-dir-request-landing'),
-        path('secure-dir-request/',
+        path('secure-dir-request/<str:project_name>',
```

Comment:
I think we're using PK in most, if not all, cases. It may make sense to move this particular URL under `project.urls`:

`/project/23/secure-dir-request`

Reply:
Got it. I didn't know if it was ok to put paths that used views from allocation inside the project urls.py